pkgs/os-specific/darwin/apple-sdk/default.nix

```diff
@@ -11,14 +11,14 @@ let
     pname = "MacOS_SDK";
     inherit version;
 
-    # This URL comes from https://swscan.apple.com/content/catalogs/others/index-10.12.merged-1.sucatalog, which we found by:
-    #  1. Google: site:swscan.apple.com and look for a name that seems appropriate for your version
+    # This URL comes from https://swscan.apple.com/content/catalogs/others/index-10.13-1.sucatalog.gz, which we found by:
+    #  1. DuckDuckGo: site:swscan.apple.com and DevSDK .pkg
```

Comment:
😆 👍

Reply:
Google gave me 1 result for the domain and DuckDuckGo has plenty to work with.